appeals/api/src/server/endpoints/lpa-questionnaires/__tests__/lpa-questionnaires.test.js

```diff
@@ -273,7 +273,7 @@ describe('lpa questionnaires routes', () => {
 
 				const body = {
 					incompleteReasons: [1, 2, 3],
-					lpaQuestionnaireDueDate: '2023-06-21',
+					lpaQuestionnaireDueDate: '2099-06-22',
```

Comment:
Perhaps we could have an additional test to check that the correct error message is returned if `lpaQuestionnaireDueDate` in the request body is in the past

Reply:
Done